src/utils/common.ts

```diff
@@ -45,7 +45,10 @@ export const formatAmountDisplay = (
     return smallAmount;
   }
 
-  return `${leftSymbol ?? ''}${new Intl.NumberFormat('en-US', { maximumFractionDigits: 2, minimumFractionDigits }).format(+amount)}`;
+  return `${leftSymbol ?? ''}${new Intl.NumberFormat('en-US', {
+    maximumFractionDigits: 2,
+    minimumFractionDigits,
+  }).format(+amount)}`;
```

Comment:
I guess just add `{ maximumFractionDigits: 2,  minimumFractionDigits }` as `const formatConfig` before for better visibility if u really wanna do this